### DISCUSSION TITLE
How to use load balancer

### DISCUSSION THREAD

**Discussion by westice007** [hnimminh/libresbc#130] [Q&A] [answered]
I found in the project description that libresbc can be used as a load balancer.
So how to use libresbc's load balancing function to form an sbc cluster server, can you give some tips? Thank you very much

**Comment by Jakihiko** [comment, marked as answer]
LB function is apply for group of gateway with outbound interconnect. Example you have 3 endpoint in EU: FR/DE/UK
you need config:
- 1 gateway for FR, 1 gateway for DE and 1 gateway for UK
- outbound interconnection with name EU, and engage 3 above gateway to this interconnection with field `weight` & `distribution= weight_based `

**Reply by westice007** [reply]
nice idea
